### PR TITLE
[27] create mobile registration screen

### PR DESCRIPTION
**Pull request by MilenaTsymbal**

### CODE REVIEW

**Review by Ivanov05Yaroslav — APPROVED** [review]
sehr gut

File: packages/mobile/app/src/main/java/com/devhub/devhubapp/LineFragment.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import android.view.View
+import android.view.ViewGroup
+
+class LineFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
```
Comment (ShutSasha) [inline packages/mobile/app/src/main/java/com/devhub/devhubapp/LineFragment.kt:15]: What is that?

File: packages/mobile/app/src/main/java/com/devhub/devhubapp/RegistrationContainerFragment.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+
+class RegistrationContainerFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
```
Comment (ShutSasha) [inline packages/mobile/app/src/main/java/com/devhub/devhubapp/RegistrationContainerFragment.kt:18]: if it not mandatory comment - remove

File: packages/mobile/app/src/main/java/com/devhub/devhubapp/RegistrationContainerFragment.kt [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        emailInputFragment.setInputHint("Enter your email")
+        fragmentTransaction.add(R.id.email_input_container, emailInputFragment)
+
+        val passwordInputFragment = InputFragment()
+        passwordInputFragment.setInputType(InputType.TYPE_TEXT_VARIATION_PASSWORD)
+        passwordInputFragment.setInputHint("Enter your password")
+        fragmentTransaction.add(R.id.password_input_container, passwordInputFragment)
+
+        val primaryButtonFragment = PrimaryButtonFragment()
+        primaryButtonFragment.setButtonText("next")
```
Comment (ShutSasha) [inline packages/mobile/app/src/main/java/com/devhub/devhubapp/RegistrationContainerFragment.kt:45]: Are u sure that "next" from lowerCase?

File: packages/mobile/app/src/main/java/com/devhub/devhubapp/TextWithLinkFragment.kt [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    private lateinit var linkTextView: TextView
+    private var text: String? = null
+    private var linkText: String? = null
+
+    @SuppressLint("MissingInflatedId")
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
```
Comment (ShutSasha) [inline packages/mobile/app/src/main/java/com/devhub/devhubapp/TextWithLinkFragment.kt:27]: same

File: packages/mobile/app/src/main/res/drawable/devhub_logo.xml
Comment (ShutSasha) [inline packages/mobile/app/src/main/res/drawable/devhub_logo.xml:1]: icons to other folder

File: packages/mobile/app/src/main/res/values/themes.xml
Comment (ShutSasha) [inline packages/mobile/app/src/main/res/values/themes.xml:1]: comments!!!!

**Review by ShutSasha — COMMENTED** [review]
before **PR** - fix issues

**Comment by MilenaTsymbal** [comment]
fixed @ShutSasha

**Pull request merged**